src/utils/getTabDetails.test.ts

```diff
@@ -268,6 +269,60 @@ describe("getNextHearingLocationExceptionsDetails", () => {
   })
 })
 
+describe("getOffencesMatchedExceptionsDetails", () => {
+  it.each([[HO100310]])(
+    "Should return ExceptionsCount:2, ExceptionsResolved:false when either of offence matching exception is raised",
```

Comment:
Is it possible to have more than 2 offence-matching exceptions?

Reply:
Yes it is, we will be brining more of them in the future tickets so it will be easier to add them in parameterised test.